lib/pages/new_group/new_group.dart

```diff
@@ -27,11 +31,15 @@ class NewGroupController extends State<NewGroup> with ComparablePresentationCont
   final fetchContactsController = FetchContactsController();
   final contactStreamController = StreamController<Either<Failure, GetContactsSuccess>>();
   final groupNameTextEditingController = TextEditingController();
+  final createRoomStreamController = CreateRoomController();
 
   final selectedContactsMapNotifier = ValueNotifier<Map<PresentationContact, bool>>({});
   final haveSelectedContactsNotifier = ValueNotifier(false);
   final haveGroupNameNotifier = ValueNotifier(false);
   final isEnableEEEncryptionNotifier = ValueNotifier(true);
+  final avatarNotifier = ValueNotifier<MatrixFile?>(null);
+
+  final int maxAvatarSize = 5 * 1000 * 1000;
```

Comment:
What is it? Why not get the `limitation` from homeserver?

Reply:
Couldn't find the way to get it from homeserver, it's called max_file_size in spec files but don't know how to get it

Reply:
`_matrix/media/v3/config`

Reply:
`api.dart`

<img width="737" alt="image" src="https://github.com/linagora/twake-on-matrix/assets/6462404/47b268ef-f0f9-4d2f-8013-5552e18b6d51">

Reply:
Thank you I used that